### DISCUSSION TITLE
SSL Authenticate

### DISCUSSION THREAD

**Discussion by Harshit1114** [rcl-ssl/docs#23] [Q&A] [answered]
I have created the SSL Certificate and import that SSL on IIS but we face this issue.[](URL![image](https://user-images.githubusercontent.com/60996494/141933185-c8f4019f-74db-419f-8afe-cf94a0b38e1f.png)

**Comment by rcladmin** [comment, marked as answer]
Try re-creating the certificate. If you can't login to the RCL Client for Let's Encrypt portal , you may need to delete your subscription in the Azure portal and re-subscribe.
